src/Button/BaseButton.js

```diff
@@ -107,18 +103,14 @@ const Button = React.forwardRef((props, ref) => {
   return <StyledButton {...props} ref={ref} />;
 });
 
-StyledButton.defaultProps = {
-  theme,
-};
-
 Button.defaultProps = {
   // base props
   // variant: 'primary',
   // size: 'medium',
   position: 'relative',
   height: '3rem',
   minWidth: '3rem',
-  m: 0,
+  // m: 0,
```

Comment:
I think we should delete this if we're not using it.

Reply:
Agreed. I'll remove them.